beta/do-app-platform-op-cli/README.md

```diff
@@ -0,0 +1,145 @@
+# [Beta] Deploy 1Password SCIM bridge on DigitalOcean App Platform with 1Password CLI
+
+This deployment example describes how to deploy 1Password SCIM bridge as an app on DigitalOcean's [App Platform](https://docs.digitalocean.com/products/app-platform/) service using [1Password CLI](https://developer.1password.com/docs/cli), the DigitalOcean command line interface ([`doctl`](https://docs.digitalocean.com/reference/doctl/)), and the DigitalOcean [1Password Shell Plugin](https://developer.1password.com/docs/cli/shell-plugins/).
```

Comment:
You can link right to the doctl shell plugin: https://developer.1password.com/docs/cli/shell-plugins/digitalocean

Reply:
I actually linked to the full page on purpose in this section to show off _all_ the plugins, and include the link directly to the DO shell plugin below:
https://github.com/1Password/scim-examples/blob/5b7f1c95da98e4bbe93415323d564a6a408b81a2/beta/do-app-platform-op-cli/README.md?plain=1#L55

I can revert this if you think it's confusing.

Reply:
Nope, that makes sense